src/powershell/FinOpsToolkit.psm1

```diff
@@ -0,0 +1,274 @@
+data LocalizedData
+{
+    # culture="en-US"
+    ConvertFrom-StringData -StringData @'
+    DeployFinOpsHub = Deploying hub from template '{0}' to resource group '{1}'.
+    ExpandingZip = Expanding zip '{0}'.
+    FoundAsset = Found asset '{0}'.
+    FoundLatestRelease = Found latest release '{0}'.
+    NewDirectory = Creating directory '{0}'.
+    NotLoggedIn = Not logged into Azure. Launching 'Login-AzAccount'.
+    VersionNotFound = Could not find version '{0}' of FinOpsHub. Run 'Get-FinOpsHubVersions' for available versions.
+'@
+}
+
+<#
+    .SYNOPSIS
+        Gets Azure context for logged in user. Will prompt for login if not currently logged in.
+#>
+function Assert-AzContext
+{
+    [CmdletBinding()]
+    param()
+    $context = Get-AzContext
+    if ($null -eq $context)
+    {
+        Write-Warning -Message ($LocalizedData.NotLoggedIn)
+        $null = Login-AzAccount
+        $context = Get-AzContext
+    }
+
+    Write-Verbose -Message ($LocalizedData.LoggedIn -f $context.Subscription.Name, $context.Subscription.Id)
+    return $context
+}
+
+function New-Directory
+{
+    [CmdletBinding()]
+    param
+    (
+        [Parameter(Mandatory = $true)]
+        [string]
+        $Path
+    )
+
+    if (-not (Test-Path -Path $Path))
+    {
+        Write-Verbose -Message ($LocalizedData.NewDirectory -f $Destination)
+        $null = New-Item -ItemType 'Directory' -Path $Destination
+    }
+}
+
+<#
+    .SYNOPSIS
+        Retrieves available version numbers of the FinOps Hub template.
+        
+    .PARAMETER Latest
+        Will only return the latest version number of FinOps Hub template.
+        
+    .EXAMPLE
+        Get-FinOpsHubVersions
+        
+        Returns all available version numbers of FinOps Hub templates.
+        
+    .EXAMPLE
+        Get-FinOpsHubVersions -Latest
+        
+        Returns only the latest version number of the FinOps Hub templates.
+#>
+function Get-FinOpsHubVersions
+{
+    [CmdletBinding()]
+    param
+    (
+        [Parameter()]
+        [switch]
+        $Latest
+    )
+    
+    # url for testing
+    $releaseUri = 'https://api.github.com/repos/dsccommunity/xdhcpserver/releases'
```

Comment:
Update to the correct repo. Should we save the repo name as a variable for the module?  
```suggestion
    $releaseUri = 'https://api.github.com/repos/microsoft/cloud-hubs/releases'
```

Reply:
done